data-visualization/data-visualization.md

```diff
@@ -25,34 +25,34 @@ Companies like [Fidelity](https://www.fidelity.com/) and [Strava](https://www.st
 
 **Participants will be able to:**:
 
-- Become more familiar with publically available data sets
-- Gain some experience determining how to select data to visualize
-- Learn about several common tools and open-source libraries for visualization
-- Utilize one (or several) tools to display data in a compelling and meaningful way
-- Use D3 methods in JavaScript to create DOM elements
-- Learn about SVG and its elements
+- Become more familiar with publically available data sets.
```

Comment:
I don't know if I agree with this change... these aren't actually complete sentences.  It's ok if bulleted lists have no punctuation, i just has to be consistent either way.  

Please change these back. Liens 37 & 38 too, please.